integrationos-api/src/endpoints/connection_definition.rs

```diff
@@ -57,6 +57,7 @@ pub struct CreateRequest {
     pub helper_link: Option<String>,
     pub authentication: Vec<AuthenticationItem>,
     pub auth_method: Option<AuthMethod>,
+    pub multi_env: Option<bool>,
```

Comment:
I think there is no need to set this as Option, you can set it as bool and then give it a false default value, that would make the data structure simpler.